src/components/Datagrid/customCells/cells.ts

```diff
@@ -105,6 +113,43 @@ export function moneyCell(
       kind: "money-cell",
       value,
       currency,
+      undiscounted,
+      locale,
+    },
+    copyData: value?.toString() ?? "",
+  };
+}
+
+interface MoneyDiscountedCellData {
+  value: number | string | null;
+  discount?: string | number;
+  undiscounted?: string | number;
+  currency: string;
+  locale: Locale;
+  lineItemId?: string;
+}
```

Comment:
**suggestion:**
also, what is lineItemId for?
```suggestion
interface MoneyDiscountedCellData extends MoneyCellData {
  lineItemId?: string;
}
```

Reply:
LineItemId is product id added to draft order